test/zone-spec/long-stack-trace-zone.spec.ts

```diff
@@ -91,6 +92,25 @@ describe('longStackTraceZone', function() {
       }, 0);
     });
   });
+
+  it('should produce long stack traces when has handled error in promise', function(done) {
+    lstz.runGuarded(function() {
+      setTimeout(function() {
+        setTimeout(function() {
+          let promise = new Promise(function(resolve, reject) {
+            setTimeout(function() {
+              reject(new Error('Hello Promise'));
+            }, 0);
+          });
+          promise.catch(function(error) {
+            expect(longStackTraceZoneSpec.getLongStackTrace(error).split('Elapsed: ').length)
+                .toBe(4);
```

Comment:
Here the asserting means with the longStackTraceZoneSpec.getLongStackTrace method, error which was catch in handing promise rejection can have long stack trace, so here I just assert the stack trace contains `Elapsed:` string.